src/main/proto/eventtemplates/video_event_template.proto

```diff
@@ -30,15 +30,17 @@ message VideoEventTemplate {
 
   // Measures the percentage of a video ad that was viewed.
   message DigitalVideoCompletionStatus {
-    uint32 completed_0_percent_plus = 1;
-
-    uint32 completed_25_percent = 2;
-
-    uint32 completed_50_percent = 3;
-
-    uint32 completed_75_percent = 4;
-
-    uint32 completed_100_percent = 5;
+    option (.wfa.measurement.api.v2aplha.event_field) = {
+      display_name = "DigitalVideoCompletionStatus"
+    };
```

Comment:
apparently there is a lint error here. Can you please have a look at the checks and fix them?